pkg/packageinstall/packageinstall.go

```diff
@@ -97,6 +97,10 @@ func (pi *PackageInstallCR) reconcile(modelStatus *reconciler.Status) (reconcile
 		return reconcile.Result{Requeue: true}, err
 	}
 
+	if pkg.Spec.Revoked.Reason != "" {
```

Comment:
again just vaguely arguing that `if pkg.Spec.Revoked.Revoked` is vaguely easier to read than `if ... != ""`